cron/internal/format/json.go

```diff
@@ -46,13 +47,13 @@ type jsonCheckDocumentationV2 struct {
 	// Can be extended if needed.
 }
 
-//nolint:govet
 type jsonCheckResultV2 struct {
-	Details []string                 `json:"details"`
-	Score   int                      `json:"score"`
+	Doc     jsonCheckDocumentationV2 `json:"documentation"`
 	Reason  string                   `json:"reason"`
 	Name    string                   `json:"name"`
-	Doc     jsonCheckDocumentationV2 `json:"documentation"`
+	Details []string                 `json:"details"`
+	ID      uint                     `json:"id"`
```

Comment:
this requires coordinating a schema update for the bigquery table. which is more involved than I have time for this week with OSS NA prep.